### DISCUSSION TITLE
When are the 500+ open PRs processed?

### DISCUSSION THREAD

**Discussion by stefandevo** [shadcn-ui/ui#4547] [General]
This is a very successful library/project but it seems like it is not maintained in a sense that there are more than 500 open PRs. I thought the idea of open source was also contributing to the code but if all these contributions are not processed I am wondering what the idea of the project is?

**Comment by stefandevo** [comment]
604.. and counting 🫣

**Reply by snovik** [reply]
I support the previous message, is the library alive?
